rules-tests/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector/Fixture/skip_error_type.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector\Fixture;
+
+use Nette\Utils\Strings;
+
+final class SkipErrorType
+{
+    public function run($regex)
+    {
+        Strings::replace('value', $regex, function (array $match) use (
+            &$string
+        ): string {
+            $innerPattern = $match['content'];
+            $positionDelimiter = strpos($innerPattern, $this->delimiter);
```

Comment:
@TomasVotruba here `$innerPattern` detected as `ErrorType` which obvious for no context ArrayDimFetch, thats' why it need to be skipped :)